api/src/queries/project/project-delete-queries.ts

```diff
@@ -3,40 +3,6 @@ import { getLogger } from '../../utils/logger';
 
 const defaultLog = getLogger('queries/project/project-delete-queries');
 
-/**
- * SQL query to delete project region rows
- *
- * @param {projectId} projectId
- * @returns {SQLStatement} sql query object
- */
-export const deleteRegionsSQL = (projectId: number): SQLStatement | null => {
-  defaultLog.debug({
-    label: 'deleteRegionsSQL',
-    message: 'params',
-    projectId
-  });
-
-  if (!projectId) {
-    return null;
-  }
-
-  const sqlStatement: SQLStatement = SQL`
```

Comment:
just to confirm, the intent is the delete **all** regions associated with a project? if the intent is to only delete a particular region associated with a project then use project_region_id to identify the record to delete.

Reply:
i guess we should get in the habit of checking users association with the data and their roles as well?

Reply:
This particular change is just showing that shreyas has deleted this whole file.  Since we plan to infer the region based on the boundary, we dont need to store region information anymore, and so the region table can be removed and all associated code, which is a large portion of the changes in this PR.